### PR TITLE
Update to geb plugin 5.0.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by jamesfredley**

### CODE REVIEW

File: examples/grails3-hibernate5/build.gradle [hunk 1]
```diff
@@ -33,7 +33,6 @@ dependencies {
 
     testImplementation("org.grails.plugins:geb") {
         exclude group: 'org.gebish', module: 'geb-spock'
-        exclude group: 'org.codehaus.groovy'
     }
```
Comment (matrei) [inline examples/grails3-hibernate5/build.gradle]: Why is `org.gebish:geb-spock` excluded and then added below? (I know it was like this before this PR)

Actually, we don't even need `org.grails.plugins:geb` here. Keep `org.gebish:geb-spock` instead.

`org.gebish:geb-spock` can be narrowed to `integrationTestImplementation`.

All the selenium modules can be narrowed to `integrationTestRuntimeOnly`.

File: examples/grails3-multiple-datasources/build.gradle [hunk 2]
```diff
@@ -25,7 +25,6 @@ dependencies {
     testImplementation "org.grails:grails-gorm-testing-support:$testingSupportVersion"
     testImplementation("org.grails.plugins:geb") {
         exclude group: 'org.gebish', module: 'geb-spock'
```
Comment (matrei) [inline examples/grails3-multiple-datasources/build.gradle]: Why is `org.gebish:geb-spock` excluded and then added below? (I know it was like this before this PR)

Actually, we don't even need `org.grails.plugins:geb` here. Keep `org.gebish:geb-spock` instead.

`org.gebish:geb-spock` can be narrowed to `integrationTestImplementation`.

All the selenium modules can be narrowed to `integrationTestRuntimeOnly`.

File: examples/grails3-schema-per-tenant/build.gradle [hunk 3]
```diff
@@ -23,9 +23,7 @@ dependencies {
     runtimeOnly "org.grails.plugins:scaffolding:$scaffoldingVersion"
 
     testImplementation "org.grails:grails-gorm-testing-support:$testingSupportVersion"
-    testImplementation "org.grails.plugins:geb:$gebPluginVersion", {
-        exclude group: 'org.codehaus.groovy'
-    }
+    testImplementation "org.grails.plugins:geb:$gebPluginVersion"
```
Comment (matrei) [inline examples/grails3-schema-per-tenant/build.gradle]: Use `org.gebish:geb-spock` instead.

`org.gebish:geb-spock` can be narrowed to `integrationTestImplementation`.

All the selenium modules can be narrowed to `integrationTestRuntimeOnly`.

**Comment by matrei** [comment]
~~I'm gonna look a bit more on the dependencies before you merge this.~~
Carry on!

File: examples/grails3-hibernate5/build.gradle (hunk 1 above)
Comment (jamesfredley) [reply examples/grails3-hibernate5/build.gradle]: There is no telling why, but I am always glad to clean them up, narrow scope and remove.

**Review by matrei — APPROVED** [review]

**Pull request merged**